src/Explorer/Tabs/QueryTab/QueryTabComponent.tsx

```diff
@@ -544,7 +544,7 @@ export default class QueryTabComponent extends React.Component<IQueryTabComponen
     useCommandBar.getState().setContextButtons(this.getTabsButtons());
   }
 
-  public setEditorContent(): string {
+  public getEditorContent(): string {
```

Comment:
I was getting confused by this method name since it _seems_ to be a `get` method rather than a `set` method, so I went ahead and renamed it. Happy to revert if I'm misunderstanding!